Terran-One/cw-simulate-ui
# Use Cloudflare Functions to load sample contract

## Issue link

[WL-XXX](https://terran-one.atlassian.net/browse/WL-XXX)

## Description

Fix load sample contract by using Cloudflare Functions as a proxy with R2 bindings. 

## Test steps

1. Pick Terra Chain and select TerraSwap from the dropdown
2. Verify that all terra-swap contracts are loaded 
3.
